src/App.js

```diff
@@ -7,26 +7,26 @@ import TeamPage from "./Pages/TeamPage";
 import MatchupPage from "./Pages/MatchupPage";
 import FreeAgentsPage from "./Pages/FreeAgentsPage";
 import LeaguePage from "./Pages/LeaguePage";
+import H2P from './Pages/H2P';
+import NBALiveStats from './Pages/NBALiveStats';
+import LandingPage from './Pages/LandingPage';
+import NHLLiveStats from './Pages/NHLLiveStats';
+
 
 function App() {
     return (
         <div className="App">
             <HashRouter>
                 <Routes>
-                    <Route exact path="/" element={<Home />} />
+                    <Route exact path="/" element={<LandingPage />} />
+                    <Route path='/nba' element={<NBALiveStats />} />
```

Comment:
make a more clear route url

Reply:
not like that on mine. Home is /home and LandingPage is /. Not sure why it's like that but I'll check it out